x-pack/plugins/ml/server/lib/alerts/alerting_service.ts

```diff
@@ -627,108 +706,32 @@ export function alertingServiceProvider(
-
-    /**
-     * Disabled until Anomaly Explorer page is fixed and properly
-     * support single point time selection
-     */
-    const highlightSwimLaneSelection = false;
-
-    const globalState = {
-      ml: {
-        jobIds: r.jobIds,
-      },
-      time: {
-        from: r.bucketRange.start,
-        to: r.bucketRange.end,
-        mode: 'absolute',
-      },
-    };
-
-    const appState = {
-      explorer: {
-        mlExplorerFilter: {
-          ...(isInfluencerResult
-            ? {
-                filterActive: true,
-                filteredFields: [
-                  r.topInfluencers![0].influencer_field_name,
-                  r.topInfluencers![0].influencer_field_value,
-                ],
-                influencersFilterQuery: {
-                  bool: {
-                    minimum_should_match: 1,
-                    should: [
-                      {
-                        match_phrase: {
-                          [r.topInfluencers![0].influencer_field_name]:
-                            r.topInfluencers![0].influencer_field_value,
-                        },
-                      },
-                    ],
-                  },
-                },
-                queryString: `${r.topInfluencers![0].influencer_field_name}:"${
-                  r.topInfluencers![0].influencer_field_value
-                }"`,
-              }
-            : {}),
-        },
-        mlExplorerSwimlane: {
-          ...(highlightSwimLaneSelection
-            ? {
-                selectedLanes: [
-                  isInfluencerResult ? r.topInfluencers![0].influencer_field_value : 'Overall',
-                ],
-                selectedTimes: r.timestampEpoch,
-                selectedType: isInfluencerResult ? 'viewBy' : 'overall',
-                ...(isInfluencerResult
-                  ? { viewByFieldName: r.topInfluencers![0].influencer_field_name }
-                  : {}),
-                ...(isInfluencerResult ? {} : { showTopFieldValues: true }),
-              }
-            : {}),
-        },
-      },
-    };
-    return `/app/ml/explorer/?_g=${encodeURIComponent(
-      rison.encode(globalState)
-    )}&_a=${encodeURIComponent(rison.encode(appState))}`;
-  };
-
   return {
     /**
      * Return the result of an alert condition execution.
      *
      * @param params - Alert params
-     * @param startedAt
-     * @param previousStartedAt
      */
     execute: async (
-      params: MlAnomalyDetectionAlertParams,
-      startedAt: Date,
-      previousStartedAt: Date | null
-    ): Promise<AnomalyDetectionAlertContext | undefined> => {
+      params: MlAnomalyDetectionAlertParams
+    ): Promise<{ context: AnomalyDetectionAlertContext; name: string } | undefined> => {
       const result = await fetchResult(params);
 
       if (!result) return;
 
-      const anomalyExplorerUrl = buildExplorerUrl(result, params.resultType);
+      const anomalyExplorerUrl = buildExplorerUrl(
```

Comment:
For the recovered context, the start time seems to be the time the rule executed, so I end up with the Anomaly Explorer opening over a very narrow time range. Should the start time by `rule execution time - lookback interval` ?

Reply:
Updated in https://github.com/elastic/kibana/pull/132496/commits/e5ab471f4eff7661212dc52d33322ba15554eb12